projects/showcase/cypress/support/e2e-driver.ts

```diff
@@ -1,32 +1,41 @@
-import { PNG } from "pngjs";
-import { MapLibreAssertable, MaplibreCypressHelper } from "./maplibre-helper";
+import { PNG } from 'pngjs';
+import { MapLibreAssertable, MaplibreCypressHelper } from './maplibre-helper';
 
 export class E2eDriver {
   private helper = new MaplibreCypressHelper();
 
   beforeAndAfter = () => {
-    beforeEach(() => {});
-    afterEach(() => {});
+    this.helper.beforeAndAfter();
+    beforeEach(() => {
+      this.helper.given.spyOnWindowConsoleError();
+      this.helper.given.spyOnWindowConsoleWarning();
+    });
+
+    afterEach(() => {
+      const then = this.then;
+      then(this.helper.get.windowConsoleWarningSpy()).shouldNotHaveBeenCalled();
```

Comment:
This looks great, are we sure this is working though?

Reply:
Yes! I checked it thoroughly